client/blocks/reader-full-post/header.jsx

```diff
@@ -43,6 +43,7 @@ const ReaderFullPostHeader = ( { post } ) => {
 					</h1>
 				</AutoDirection>
 			) : null }
+			<div className="reader-full-post__author-block">{ authorProfile }</div>
```

Comment:
Added a new class name here to target to prevent the new CSS from leaking / causing regressions elsewhere.